core/include/traccc/fitting/kalman_filter/kalman_actor.hpp

```diff
@@ -114,6 +114,9 @@ struct kalman_actor : detray::actor {
                 return;
             }
 
+            // Update the propagation flow
+            propagation._stepping.bound_params() = trk_state.filtered();
```

Comment:
how about `stepping.bound_params() = trk_state.filtered()` after removing the `const` qualifier above?

Reply:
Sure, done